board/microchip/polarberry/post-image.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+#Create FAT linux Image (linux kernel  + rootfs + DTS)
+cp -R output/images ${BR2_EXTERNAL_MCHP_PATH}/board/microchip/polarberry
```

Comment:
why are you copying the entire image directory here? you should not do this. Please remove this line.